.github/workflows/php.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2.0.0
       - name: Run PHP-CS-Fixer
-        uses: prestashopcorp/github-action-php-cs-fixer@master
+        uses: sowbiba/github-action-php-cs-fixer@use-php-dev-tools-master
```

Comment:
Is that wanted ?

Reply:
Just testing because we had a warning

Reply:
https://github.com/PrestaShopCorp/github-action-php-cs-fixer/pull/2/files